src/main/java/domain/Users.java

```diff
@@ -0,0 +1,36 @@
+package domain;
+
+import exception.ErrorMessage;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Users {
+    private final List<User> users;
+
+    public Users(final List<String> userNames) {
+        validateUserNamesEmpty(userNames);
+        users = new ArrayList<>();
+        for (final String userName : userNames) {
+            users.add(new User(userName));
+        }
```

Comment:
요런 경우 initUsers() 메서드로 만들어보면 깔끔할것 같네요
```suggestion
        this.users = initUsers(userNames);
```

Reply:
생성자에 많은 코드가 들어가 있는 것으로 여겨질 수 있어 메서드 분리를 하는 것도 좋은 방법인 것 같습니다.